lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board
+	attr_accessor :cells
+	
+	def initialize
+		@cells = Array.new(9," ")

```

Comment:
Great way to initialize the array, but isn't this code already in `#reset!`? You could just call the method here.